roboto/bot.py

```diff
@@ -735,6 +736,53 @@ async def stop_inline_message_live_location(
             ),
         )
 
+    async def send_venue(
+        self,
+        chat_id: Union[ChatID, str],
+        latitude: float,
+        longitude: float,
+        title: str,
+        address: str,
+        foursquare_id: Optional[str] = None,
+        foursquare_type: Optional[str] = None,
+        disable_notification: Optional[bool] = None,
+        reply_to_message_id: Optional[MessageID] = None,
+        reply_markup: Optional[ReplyMarkup] = None,
+    ) -> Message:
+        """sendVenue API method.
+
+        Args:
+            chat_id: The ID of the chat to send a venue to.
+            latitude: Latitude of the venue.
+            longitude: Longitude of the venue.
+            title: Name of the venue.
+            address: Address of the venue.
+            foursquare_id: Foursquare identifier of the venue.
+            foursquare_type: Foursquare type of the venue, if known.
+            disable_notification: Do not notify users that the message was sent.
+            reply_to_message_id: ID of a message that the sent message should
+                                 be a reply to.
+            reply_markup: Markup for additional interface options for replying.
+        """
+
+        request = SendVenueRequest(
+            chat_id,
+            latitude,
+            longitude,
+            title,
+            address,
+            foursquare_id,
+            foursquare_type,
+            disable_notification,
+            reply_to_message_id,
+            maybe_json_serialize(reply_markup),
+        )
+
+        return from_json(
+            Message,
+            await make_request(self.session, HTTPMethod.POST, '/sendVenue', request),
```

Comment:
Isn't it a bit weird to have a parameter of `make_request` be `request`? Or since we are dealling with assyncronous code it is clear that `await make_request(... request)` means "wait for the request to be executed"?